voting_schemes/electionguard/python-wrapper/src/bulletin_board/electionguard/serializable.py

```diff
@@ -59,27 +60,27 @@ def maybe_base64_to_int(i: Union[str, int]) -> int:
 
 
 def set_serializers():
-    old_set_serializers()
+    # old_set_serializers()
     electionguard.serializable.set_serializer(
-        lambda p, **_: int_to_maybe_base64(p.to_int()), ElementModP
+        lambda p, **_: int_to_maybe_base64(p.to_int()), ElementModP  # type: ignore
     )
     electionguard.serializable.set_serializer(
-        lambda q, **_: int_to_maybe_base64(q.to_int()), ElementModQ
+        lambda q, **_: int_to_maybe_base64(q.to_int()), ElementModQ  # type: ignore
     )
     electionguard.serializable.set_serializer(
-        lambda i, **_: int_to_maybe_base64(i), int
+        lambda i, **_: int_to_maybe_base64(i), int  # type: ignore
     )
 
 
 def set_deserializers():
-    old_set_serializers()
+    # old_set_serializers()
```

Comment:
⚠️ same as before. Also, the method name is wrong, so probably it was not doing anything 🙈

Reply:
Kept it~